web/docs/project/env-vars.md

```diff
@@ -86,13 +190,11 @@ In the root of your Wasp project you can create two distinct files:
     REACT_APP_SOME_VAR_NAME=somevalue
     ```
 
+    <ClientEnvVarsNote />
+
 `.env.server` should not be committed to version control as it can contain secrets, while `.env.client` can be versioned as it must not contain any secrets.
 By default, in the `.gitignore` file that comes with a new Wasp app, we ignore all dotenv files.
 
-:::info Dotenv files
```

Comment:
Where did this go, it wasn't useful?

Reply:
It didn't feel useful, it talks about the specific Node.js package and they are not going to use it directly. I think the env vars file format is self-explanatory.

Reply:
They might need to understand how to use it though, to for example pass JSON values and similar, we had those situations, how to quote stuff. So I thoguht it is good to introduce the concept, since we are using it as na important part of Wasp, so it is clear this is not something we made up, but instead there are more resources behind it that they can look for. Drop it if you really want, but to me it seems useful to keep it.

Reply:
I get what you mean, if people know what's underneath they can Google better. 

The env variables are first parsed by Haskell and then outputted to the `.env` files in the server and the client apps. If we wanted to provide full information, we should first mention Haskell's [dotenv](https://hackage.haskell.org/package/dotenv) package and then our logic for outputting the vars i.e. our `envVarsToDotEnvContent` function. And then we can mention that the variables are ingested by Node.js with the JS [dotenv](https://www.npmjs.com/package/dotenv) package.

I dislike giving this many details to users because 99% of them don't really need them and those with issues will most likely ask about it in our Discord.